### DISCUSSION TITLE
How to update data in cheatsheet?

### DISCUSSION THREAD

**Discussion by nullptroma** [end-4/dots-hyprland#658] [Q&A]
I made my keybind settings based on those that were originally there. But still they differ quite a lot. How do I regenerate the settings for cheat sheet?

**Comment by evangelosommer** [comment]
If I understand your question correctly, when using the `~/.config/hypr/custom/keybinds.conf` file for your custom keybinds (which is recommended), ensure that you have updated the path within `~/.config/ags/modules/cheatsheet/keybinds.js` to point to you custom keybind configurations. Otherwise, the cheatsheet will only load from the default `~/.config/hypr/hyprland/keymaps.conf` file path.
